app/models/order.rb

```diff
@@ -2,4 +2,5 @@ class Order < ApplicationRecord
   belongs_to :guest
   belongs_to :user
   has_many :order_details
+  validates :status, presence: true
```

Comment:
Có nhưng loại status nào:
thêm validate cho những status đó như sau
validates :status, inclusion: { in: %w(pending delivered finish) }

Reply:
ok, cái này hay :v